frontend/public/components/edit-yaml.jsx

```diff
@@ -356,7 +356,7 @@ export const EditYAML = connect(stateToProps)(
         { canDrop && <div className={klass}><p className="co-file-dropzone__drop-text">Drop file here</p></div> }
 
         <div>
-          {create && <div className="yaml-editor__header">
+          {create && this.props.showHeader && <div className="yaml-editor__header">
```

Comment:
Introduced this so we can show the same header with breadcrumbs between form/YAML editor.